toopher/__init__.py

```diff
@@ -110,106 +122,77 @@ def validate(self, data, request_token=None, ttl=DEFAULT_IFRAME_TTL):
 
         return data
 
-    def signature(self, data):
+    def _signature(self, data):
```

Comment:
What does this method do? I know it's private-ish, but it'd be nice if it was a bit more descriptive. What kind of signature is it, for example?